src/test/java/com/ays/assignment/controller/AssignmentControllerTest.java

```diff
@@ -55,10 +56,11 @@ class AssignmentControllerTest extends AbstractRestControllerTest {
     private final AssignmentToAssignmentSearchResponseMapper assignmentToAssignmentSearchResponseMapper = AssignmentToAssignmentSearchResponseMapper.initialize();
     private final AssignmentToAssignmentResponseMapper assignmentToAssignmentResponseMapper = AssignmentToAssignmentResponseMapper.initialize();
     private final AssignmentToAssignmentsResponseMapper assignmentToAssignmentsResponseMapper = AssignmentToAssignmentsResponseMapper.initialize();
+    private final AssignmentToAssignmentSummaryResponseMapper assignmentToAssignmentSummaryResponseMapper = AssignmentToAssignmentSummaryResponseMapper.initialize();
     private final AssignmentToAssignmentUserResponseMapper assignmentToAssignmentUserResponseMapper = AssignmentToAssignmentUserResponseMapper.initialize();
 
 
-    private static final String BASE_PATH = "/api/v1";
+    private final String BASE_PATH = "/api/v1";
```

Comment:
Bu field için neden `static` keyword'ünü kaldırdık?

Reply:
Geri ekledim.